includes/class-monk-activator.php

```diff
@@ -30,7 +29,15 @@ class Monk_Activator {
 	 * @since    1.0.0
 	 */
 	public static function activate() {
+		global $monk_languages;
+		if ( $monk_languages[ get_locale() ]['native_name'] ) {
```

Comment:
Use `array_key_exists` to check if locale exists.
http://php.net/manual/pt_BR/function.array-key-exists.php